src/views/coreDam/podcast/components/PodcastEditForm.vue

```diff
@@ -66,6 +66,14 @@ const { podcastModeOptions } = usePodcastMode()
             data-cy="podcast-rss-url"
           />
         </ARow>
+        <ARow>
+          <AFormTextField
+            v-model="podcast.attributes.extUrl"
+            :label="t('coreDam.podcast.model.attributes.extUrl')"
```

Comment:
Is this `label` attribute required? I think it should be resolved from `v` attribute.

Reply:
Yes, it is possible. But each line of the form has a specified `label` (also each CRUD form), so more code modification is probably needed, but that's beyond the scope of this PR